### PR TITLE
feat: add chakra-ui-vue

### PR DESCRIPTION
**Pull request by Easybuoy**
Added `chakra-ui-vue` ui library

https://vue.chakra-ui.com

### CODE REVIEW

File: packages/create-nuxt-app/test/snapshots/index.test.js.md [hunk 1]
```diff
@@ -3223,6 +3223,118 @@ Generated by [AVA](https://avajs.dev).
...
+      */␊
+      buildModules: [␊
+      ],␊
+      /*␊
+      ** Nuxt.js modules␊
+      */␊
+      modules: [␊
+          // Doc: https://github.com/nuxt-community/emotion-module#readme␊
+          '@nuxtjs/emotion',␊
+      ],␊
```
Comment (codebender828) [inline packages/create-nuxt-app/test/snapshots/index.test.js.md:3328]: And this is `@chakra-ui/nuxt` 👍🏽

File: packages/cna-template/template/frameworks/chakra-ui/package.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "dependencies": {
+    "@chakra-ui/vue": "^0.6.0",
```
[the remark this answers is not in the document]
Comment (Easybuoy) [reply packages/cna-template/template/frameworks/chakra-ui/package.json:3]: The documentation also says we need to install the `@nuxtjs/emotion` package as well, I also tested on code sandbox, and it seemed to work smoothly with the`@nuxtjs/emotion` module added

[the remark this answers is not in the document]
Comment (codebender828) [reply packages/cna-template/template/frameworks/chakra-ui/package.json:3]: Oh! I see. Then you made the right decision. I think I might have misread the docs 😅. I was using my phone to check. Then we can add it here  👍🏽

Awesome then let's add it and make the other adjustments and we can go with this.

Really great work here! 👍🏽

[the remark this answers is not in the document]
Comment (Easybuoy) [reply packages/cna-template/template/frameworks/chakra-ui/package.json]: Thanks 👍🏽

**Pull request merged**